code/chris/feedback/lab03_v1.py

```diff
@@ -0,0 +1,44 @@
+tens_dict = {
+  2: 'twenty',
+  3: 'thirty',
+  4: 'forty',
+  5: 'fifty',
+  6: 'sixty',
+  7: 'seventy',
+  8: 'eighty',
+  9: 'ninety'
+}
+ones_dict = {
+  1: 'one',
+  2: 'two',
+  3: 'three',
+  4: 'four',
+  5: 'five',
+  6: 'six',
+  7: 'seven',
+  8: 'eight',
+  9: 'nine',
+}
+
+x = 32
+
+# run this code then comment this block out
```

Comment:
I made some alterations to test this script for 0-99 and it caused some KeyErrors.  Run this altered script.  Then comment this block out.